steam-overlay.conf

```diff
@@ -3,3 +3,4 @@ location = /usr/local/steam-overlay/
 sync-type = git
 sync-uri = https://github.com/anyc/steam-overlay/
 auto-sync = Yes
+masters = gentoo
```

Comment:
This line belongs in `metadata/layout.conf` as per developer's guidance at [https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf](url)